trustyaiexplainability-core/src/test/java/org/kie/trustyai/explainability/utils/DataUtilsTest.java

```diff
@@ -237,7 +238,7 @@ void testDoublesToFeatures() {
         for (Feature f : features) {
             assertNotNull(f);
             assertNotNull(f.getName());
-            assertEquals(Type.NUMBER, f.getType());
+            Assertions.assertEquals(Type.NUMBER, f.getType());
```

Comment:
This change should not be necessary, there is already an import static of `assertEquals`